packages/vendure-plugin-limit-variant-per-order/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.3.0 (2024-04-10)
+
+- Added support for channel aware `onlyAllowPer` value per ProductVariant
```

Comment:
I think we can maybe make this a version 2.0.0, since the customField is a breaking change